src/content/mods.cpp

```diff
@@ -55,6 +55,7 @@ bool parseDependsString(std::string &dep, std::unordered_set<char> &symbols)
 	return !dep.empty();
 }
 
+[[nodiscard]]
```

Comment:
missing annotation in `mods.h`

Reply:
fixing that will raise the question of what to do if this fails in `l_mainmenu.cpp`. it should return nothing in that case, which would then have to be dealt with properly at all callsites of `core.get_content_info`.